### PR TITLE
Accessibility review of portfolio project

### PR DESCRIPTION
**Pull request by chahine-mahdhi**
In this PR, we have made some changes to the portfolio project to ensure its accessibility.
* Page titles:
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863638590

* Image text alternatives:
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863639480
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863640991
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641048
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641120
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641203
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641285
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863643086
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863643152
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863643376

* Text headings:
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863640784
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641668
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641752
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641851
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863641953
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863642035
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863642126
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863642212

* Color contrast:
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863645141
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863645208
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863645291
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863645345

* Resize - no issues found

* Interaction:
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863643725
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863644091
https://github.com/chahine-mahdhi/portfolio/pull/9#discussion_r863644319

* Moving content - no issues found

* Multimedia - no issues found

* The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap"
       rel="stylesheet"
     />
-    <title>Portfolio</title>
+    <title>Chahine-Portfolio</title>
```
Comment (chahine-mahdhi) [inline index.html:12]: We have changed the title to make it unique and meaningful.

File: index.html [hunk 2]
```diff
@@ -24,13 +24,13 @@
...
       </nav>
-      <nav class="menu"><img src="./logos/list.png" alt="menu" /></nav>
+      <nav class="menu"><img src="./logos/list.png" alt="Clickable hamburger menu" /></nav>
     </header>
     <main>
       <section class="info-page">
         <div class="info">
           <h1 class="primary-text">Hey There.<br />I’m Chahine</h1>
-          <p class="supporting-text">I’am a Software Developer</p>
+          <h2 class="supporting-text">I’am a Software Developer</h2>
```
Comment (chahine-mahdhi) [inline index.html:33]: Change from p tag to h2 tag for more accessibility.

File: index.html [hunk 3]
```diff
@@ -39,23 +39,23 @@ <h1 class="primary-text">Hey There.<br />I’m Chahine</h1>
         </div>
         <ul class="social-media">
           <li>
-            <a href="#"><img src="./logos/github.png" alt="github-logo" /></a>
+            <a href="#"><img src="./logos/github.png" alt="Clickable github-logo" /></a>
```
Comment (chahine-mahdhi) [inline index.html:42]: We have changed the alt attribute to make it functional and provide an equivalent user experience.

File: index.html [hunk 4]
```diff
@@ -39,23 +39,23 @@ <h1 class="primary-text">Hey There.<br />I’m Chahine</h1>
...
           </li>
           <li>
             <a href="#"
-              ><img src="./logos/linkedin.png" alt="linkedin-logo"
+              ><img src="./logos/linkedin.png" alt="Clickable linkedin-logo"
```
Comment (chahine-mahdhi) [inline index.html:46]: We have changed the alt attribute to make it functional and provide an equivalent user experience.

File: index.html [hunk 5]
```diff
@@ -39,23 +39,23 @@ <h1 class="primary-text">Hey There.<br />I’m Chahine</h1>
...
             /></a>
           </li>
           <li>
             <a href="#"
-              ><img src="./logos/angelist.png" alt="angelist-logo"
+              ><img src="./logos/angelist.png" alt="Clickable angelist-logo"
```
Comment (chahine-mahdhi) [inline index.html:51]: We have changed the alt attribute to make it functional and provide an equivalent user experience.

File: index.html [hunk 6]
```diff
@@ -39,23 +39,23 @@ <h1 class="primary-text">Hey There.<br />I’m Chahine</h1>
...
             /></a>
           </li>
           <li>
-            <a href="#"><img src="./logos/twitter.png" alt="twitter-logo" /></a>
+            <a href="#"><img src="./logos/twitter.png" alt="Clickable twitter-logo" /></a>
```
Comment (chahine-mahdhi) [inline index.html:55]: We have changed the alt attribute to make it functional and provide an equivalent user experience.

File: index.html [hunk 7]
```diff
@@ -39,23 +39,23 @@ <h1 class="primary-text">Hey There.<br />I’m Chahine</h1>
...
           </li>
           <li>
-            <a href="#"><img src="./logos/medium.png" alt="medium-logo" /></a>
+            <a href="#"><img src="./logos/medium.png" alt="Clickable medium-logo" /></a>
```
Comment (chahine-mahdhi) [inline index.html:58]: We have changed the alt attribute to make it functional and provide an equivalent user experience.

File: index.html [hunk 8]
```diff
@@ -66,7 +66,7 @@ <h2 class="works-title">My Recent Works</h2>
             <img class="mobile-works" src="./logos/img-placeholder.png" alt="work example" />
             <img class="desktop-works" src="./logos/ImgPlaceholder.png" alt="work example" />
             <div class="card-content">
-              <p class="card-title">Multi-Post Stories</p>
+              <h3 class="card-title">Multi-Post Stories</h3>
```
Comment (chahine-mahdhi) [inline index.html:69]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 9]
```diff
@@ -83,7 +83,7 @@ <h2 class="works-title">My Recent Works</h2>
             </div>
           </div>
           <div class="other-card">
-            <p class="other-card-title">Professional Art Printing Data</p>
+            <h3 class="other-card-title">Professional Art Printing Data</h3>
```
Comment (chahine-mahdhi) [inline index.html:86]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 10]
```diff
@@ -96,7 +96,7 @@ <h2 class="works-title">My Recent Works</h2>
             <a class="button" href="#">See Project</a>
           </div>
           <div class="other-card">
-            <p class="other-card-title">Professional Art Printing Data</p>
+            <h3 class="other-card-title">Professional Art Printing Data</h3>
```
Comment (chahine-mahdhi) [inline index.html:99]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 11]
```diff
@@ -109,7 +109,7 @@ <h2 class="works-title">My Recent Works</h2>
...
```
Comment (chahine-mahdhi) [inline index.html:112]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 12]
```diff
@@ -122,7 +122,7 @@ <h2 class="works-title">My Recent Works</h2>
...
```
Comment (chahine-mahdhi) [inline index.html:125]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 13]
```diff
@@ -135,7 +135,7 @@ <h2 class="works-title">My Recent Works</h2>
...
```
Comment (chahine-mahdhi) [inline index.html:138]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 14]
```diff
@@ -148,7 +148,7 @@ <h2 class="works-title">My Recent Works</h2>
...
```
Comment (chahine-mahdhi) [inline index.html:151]: Change from p tag to h3 tag for more accessibility.

File: index.html [hunk 15]
```diff
@@ -179,7 +179,7 @@ <h2>About me</h2>
               <img
                 class="rectangle"
                 src="./logos/Rectangle55.png"
-                alt="rectangle"
+                alt=""
```
Comment (chahine-mahdhi) [inline index.html:182]: removed alt attribute text since it has no useful information.

File: index.html [hunk 16]
```diff
@@ -195,7 +195,7 @@ <h3 class="skill">Language</h3>
               <img
                 class="rectangle"
                 src="./logos/Rectangle56.png"
-                alt="rectangle"
+                alt=""
```
Comment (chahine-mahdhi) [inline index.html:198]: removed alt attribute text since it has no useful inforamtion.

File: index.html [hunk 17]
```diff
@@ -212,7 +212,7 @@ <h3 class="skill">Frameworks</h3>
               <img
                 class="rectangle"
                 src="./logos/Rectangle57.png"
-                alt="rectangle"
+                alt=""
```
Comment (chahine-mahdhi) [inline index.html:215]: removed alt attribute text since it has no useful information.

File: index.html [hunk 18]
```diff
@@ -233,13 +233,17 @@ <h2>
             like to chat please get in touch.
           </h2>
           <form action="https://formspree.io/f/xbjwpgpy" method="POST">
+            <label for="Full-name"></label>
```
Comment (chahine-mahdhi) [inline index.html:236]: added label tag for accessibility.

File: index.html [hunk 19]
```diff
@@ -233,13 +233,17 @@ <h2>
...
             <input
               type="text"
+              id="Full-name"
               placeholder="Full name"
               required
               maxlength="30"
             />
-            <input type="email" placeholder="Email address" required />
+            <label for="email-address"></label>
```
Comment (chahine-mahdhi) [inline index.html:244]: added label tag for accessibility.

File: index.html [hunk 20]
```diff
@@ -233,13 +233,17 @@ <h2>
...
+            <input type="email" id="email-address" placeholder="Email address" required />
+            <label for="description">
```
Comment (chahine-mahdhi) [inline index.html:246]: added label tag for accessibility.

File: style.css [hunk 21]
```diff
@@ -197,7 +197,7 @@ ul {
 
 .first-card a {
   padding: 12px;
-  background: #ff6b00;
+  background: #943e00;
```
Comment (chahine-mahdhi) [inline style.css:200]: We have changed the background color to fix the color contrast accessibility issue.

File: style.css [hunk 22]
```diff
@@ -215,7 +215,7 @@ ul {
   text-align: center;
   padding-top: 3.3%;
   padding-bottom: 3.3%;
-  background: #ff6b00;
+  background: #943e00;
```
Comment (chahine-mahdhi) [inline style.css:218]: We have changed the background color to fix the color contrast accessibility issue.

File: style.css [hunk 23]
```diff
@@ -231,7 +231,7 @@ ul {
   width: 50%;
   text-align: center;
   padding: 12px;
-  background: #ff6b00;
+  background: #943e00;
```
Comment (chahine-mahdhi) [inline style.css:234]: We have changed the background color to fix the color contrast accessibility issue.

File: style.css [hunk 24]
```diff
@@ -410,7 +410,7 @@ form button {
   margin-top: 36px;
   text-align: center;
   padding: 12px;
-  background: #ff6b00;
+  background: #943e00;
```
Comment (chahine-mahdhi) [inline style.css:413]: We have changed the background color to fix the color contrast accessibility issue.

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @chahine-mahdhi, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both members in your pair programming group. Kindly make sure both of you contribute and commit.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**